resources/monitoring/values.yaml

```diff
@@ -1779,8 +1779,9 @@ prometheus:
       matchNames:
         - istio-system
     endpoints:
-      - port: prometheus
-        interval: 30s
+# disabled to fix wrong istio-ingressgateway service port configuration
```

Comment:
please link the istio ticket in the comment and name the file to be removed after the issue got fixed